ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/loopback-interfaces.j2

```diff
@@ -47,6 +47,14 @@ interface {{ loopback_interface }}
    mpls ldp interface
 {%     elif loopback_interfaces[loopback_interface].mpls.ldp.interface is arista.avd.defined(false) %}
    no mpls ldp interface
+{%     endif  %}
+{%     if loopback_interfaces[loopback_interface].node_segment.index is arista.avd.defined %}
```

Comment:
This is already covered by a slightly different data model `node_segment.ipv4_index` and `node_segment.ipv6_index` below. Is this on purpose? If so, you need to update the documentation.

Reply:
Removed this.